python/lsst/afw/image/exposureInfo.cc

```diff
@@ -32,6 +35,7 @@ PYBIND11_PLUGIN(_exposureInfo) {
     py::module mod("_exposureInfo", "Python wrapper for afw _exposureInfo library");
 
     /* Module level */
+    py::class_<ExposureInfo, PTR(ExposureInfo)> clsExposureInfo(mod, "ExposureInfo");
```

Comment:
Yes, but copy paste was easier. I intend to run `sed` on the whole codebase at some point to remove `PTR` and `CONST_PTR` as part of DM-6056.